src/test/lib/ApiTest.cpp

```diff
@@ -5980,549 +5980,333 @@ QuicTestCredentialLoad(const QUIC_CREDENTIAL_CONFIG* Config)
     TEST_QUIC_SUCCEEDED(Configuration.LoadCredential(Config));
 }
 
+
+class QuicStorageSettingScopeGuard {
+public:
+    static
+    QuicStorageSettingScopeGuard Create(
+        _In_opt_ const char* StorageName = nullptr) {
```

Comment:
nit: The C++ style used around here seems to have the `{` on a new line, which IMO helps with not having the body with the same indentation + no line jump right after the parameters.

Reply:
(this seems to be the case in msquic.hpp, which I was following, but that file is also confused)